suites/squid/rgw/baremetal/mero_scale_rgw_multi_site.yaml

```diff
@@ -251,7 +251,49 @@ tests:
       desc: create versioned bucket in primary
       module: sanity_rgw_multisite.py
       name: create versioned bucket in primary
-      polarion-id: CEPH-83575073
+      polarion-id: CEPH-83575073 # CEPH-83575304
+
+  - test:
+      name: parallel manually resharding bucket to 32 number of shards
+      desc: parallel manually resharding bucket to 32 number of shards
+      module: test_parallel.py
+      parallel:
+        - test:
+            clusters:
+              ceph-sec:
+                config:
+                  cephadm: true
+                  commands:
+                    - "radosgw-admin bucket stats --bucket cosbench01-bkt-1"
+            desc: check bucket stats of bucket on secondary
+            name: check bucket stats of bucket on secondary
+            module: exec.py
+            polarion-id: CEPH-83575304
+        - test:
+            clusters:
+              ceph-pri:
+                config:
+                  cephadm: true
+                  commands:
+                    - "radosgw-admin bucket stats --bucket cosbench01-bkt-1"
+                    - "radosgw-admin bucket reshard --bucket cosbench01-bkt-1 --num-shards=32"
+                    - "radosgw-admin bucket stats --bucket cosbench01-bkt-1"
```

Comment:
For now changes looks good,
but since we are not validating expected value being set, 
validation needs to be added as a part of https://issues.redhat.com/browse/RHCEPHQE-18370